src/System.Xml.XmlSerializer/src/System.Xml.XmlSerializer.csproj

```diff
@@ -100,5 +100,9 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="System\Xml\Serialization\ReflectionXmlSerializationReader.cs" />
+    <Compile Include="System\Xml\Serialization\ReflectionXmlSerializationWriter.cs" />
+  </ItemGroup>
```

Comment:
Question only: these 2 new files are not in the same <ItemGroup> as all the others (see above).  Is that intentional?